SingleScanParameters.md

```diff
@@ -208,6 +210,30 @@ Enables random movement of humanoids within stalemate situation
 
 Technical information: `HumPlacement`
 
+### HasAiGeneratedLoops
```

Comment:
We should note that this parameter has a higher priority than [HasRandomLoops](https://github.com/Corticore/open-docs/blob/main/SingleScanParameters.md#HasRandomLoops).

The note should be added here and there: [HasRandomLoops](https://github.com/Corticore/open-docs/blob/main/SingleScanParameters.md#HasRandomLoops)